tests/test_security_path_traversal.py

```
@@ -187,6 +187,23 @@ async def run_tests():
 
     asyncio.run(run_tests())
 
+
+def test_stream_run_events_path_validation():
+    """Test that stream_run_events validates run_id against path traversal."""
+    import asyncio
+    from fastapi import HTTPException
+    from swarm.api.routes.events import stream_run_events
+
+    async def run_tests():
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `run_tests`

Add return type annotation: `None`

(ANN202)